en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+                FOREIGN KEY (PARENT_CUSTOMER_ID) REFERENCES AM_MONETIZATION_PLATFORM_CUSTOMERS(ID) ON DELETE CASCADE
+            );
+        
+            CREATE SEQUENCE AM_MONETIZATION_SUBSCRIPTIONS START WITH 1 INCREMENT BY 1;
+            CREATE TABLE IF NOT EXISTS AM_POLICY_SUBSCRIPTION (
+                        POLICY_ID INTEGER DEFAULT NEXTVAL('AM_MONETIZATION_SUBSCRIPTIONS'),
+                ID INTEGER NOT NULL AUTO_INCREMENT,
+                SUBSCRIBED_APPLICATION_ID INTEGER NOT NULL,
+                SUBSCRIBED_API_ID INTEGER NOT NULL,
+                TENANT_ID INTEGER NOT NULL,
+                SUBSCRIPTION_ID VARCHAR(256) NOT NULL,
+                SHARED_CUSTOMER_ID INTEGER NOT NULL,    
+                PRIMARY KEY (ID),
+                FOREIGN KEY (SUBSCRIBED_APPLICATION_ID) REFERENCES AM_APPLICATION(APPLICATION_ID) ON DELETE CASCADE,
+                FOREIGN KEY (SUBSCRIBED_API_ID) REFERENCES AM_API(API_ID) ON DELETE CASCADE,
+                FOREIGN KEY (SHARED_CUSTOMER_ID) REFERENCES AM_MONETIZATION_SHARED_CUSTOMERS(ID) ON DELETE CASCADE
+            );
+        ```
+
+    6.  Configure the tenant admin on WSO2 API Manager.
+        1.  Start the WSO2 API Manager server.
+        2.  Sign in to the WSO2 API-M management console.
+        3.  Add the following configuration in
+            `tenant-conf.json` (`/_system/config/apimgt/applicationdata/tenant-conf.json`) file using the management console.  
+
+            ``` json tab="Format"
+            "MonetizationInfo": {
+                "<key-value-pair>"
+            }
+            ```
+
+             * `<key-value-pair>` - The key value pair varies based on your monetization interface. There can be multiple key value pairs.
+
+            ``` json tab="Example"
+            "MonetizationInfo": {
+                "BillingEnginePlatformAccountKey": "sk_test_QEGKUR7yt83Bc0RdAme0bZPL00hnxaNCE6"
+            }
+            ```
+
+             * `sk_test_QEGKUR7yt83Bc0RdAme0bZPL00hnxaNCE6` - In the case of Stripe this is the platform account key that corresponds to the tenant admin, which you obtained after creating the Stripe account for tenant admin.
+
+    7. Optionally, change the granularity for data aggregation if required.
+
+        1.  Navigate to the `<API-M_HOME>/repository/conf/deployment.toml` file.
+        2.  Add the following configuration in the TOML file. 
+
+            ``` java
+            "apim.monetization.granularity": "seconds" 
+            ```
+
+##### (C.) - Configure the billing engine with WSO2 API-M Analytics
+When working with API Monetization, it is mandatory to enable WSO2 API Manager Analytics, because WSO2 API Manager needs to fetch API usage from analytics.
+
+1.  Enable WSO2 API Manager Analytics.  
+    
+     You need to enable WSO2 API Manager Analytics in order to use Monetization, because when working with Monetization the API usage is derived from WSO2 API Manager Analytics.  
```

Comment:
rather than saying "derived form WSO2 API Manager Analytics" isn't it better to say "You need to enable WSO2 API Manager Analytics in order to use Monetization, because  the API usage is obtained via  WSO2 API Manager Analytics"

Reply:
Removed the sentence altogether as the intro para to this main step has the same information.